### PR TITLE
Adds icons for Audience, Money, Platforms, TiltLogoOutline

### PR DESCRIPTION
**Pull request by BrettBukowski**
screen shot 2015-08-03 at 4 24 03 pm

### CODE REVIEW

File: app/components/Money.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            'tilt-icon',
+            'tilt-icon-money',
+            this.props.className
+        );
+
+        return (
+            <svg className={classes} viewBox="0 0 63 42">
+                <title>Money</title>
+                <defs></defs>
+                <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (tildedave) [inline app/components/Money.js]: `strokeWidth` (otherwise there is a react warning)

File: app/index.js [hunk 2]
```diff
@@ -28,5 +28,9 @@ include('TiltLogoTextImage');
 include('Triangle');
 include('TwitterLogo');
 include('UsersTwo');
+include('Money');
```
Comment (tildedave) [inline app/index.js]: alphabetize in the list (sorry)

**Comment by tildedave** [comment]
I think you want to color the icons `fill` with CSS rather than an inline style.  that'll make these more flexible in the event that you have to change the color

File: app/components/Money.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    render() {
+        const classes = classnames(
+            'tilt-icon',
+            'tilt-icon-money',
+            this.props.className
+        );
+
+        return (
+            <svg className={classes} viewBox="0 0 63 42">
+                <title>Money</title>
```
Comment (tildedave) [inline app/components/Money.js]: I generally remove title/defs but ymmv

**Comment by BrettBukowski** [comment]
thanks for the feedback @tildedave!

ping

**Comment by tildedave** [comment]
lgtm :airplane:

**Pull request merged**